iroh/src/magicsock/relay_actor.rs

```diff
@@ -72,11 +81,22 @@ struct ActiveRelayActor {
 #[allow(clippy::large_enum_variant)]
 enum ActiveRelayMessage {
     GetLastWrite(oneshot::Sender<Instant>),
-    Ping(oneshot::Sender<Result<Duration, ClientError>>),
-    GetLocalAddr(oneshot::Sender<Option<SocketAddr>>),
+    /// Returns whether or not this relay can reach the NodeId.
     GetNodeRoute(NodeId, oneshot::Sender<bool>),
```

Comment:
should this be renamed to `HasNodeRoute`?

Reply:
Thanks, I needed to do this in the last PR but didn't find a good name.  But that suggestion is great.